pages/Transfer.qml

```diff
@@ -772,7 +766,7 @@ Rectangle {
         }
 
         // There are sufficient unlocked funds available
-        if(parseFloat(amountLine.text) > parseFloat(middlePanel.unlockedBalanceText)){
+        if(parseFloat(amountLine.text) > parseFloat(walletManager.displayAmount(currentWallet.balance(currentWallet.currentSubaddressAccount)))){
             root.sendButtonWarning = qsTr("Amount is more than unlocked balance.") + translationManager.emptyString;
```

Comment:
This error message is also never displayed.

User only gets a pop up with 
`Can't create transaction: not enough money to transfer, overall balance only 0.000000000000, sent amount 1111111.000000000000`